DiscordBot/Commands/EventCommands.cs

```diff
@@ -141,6 +146,24 @@ await context.Dependencies.GetDependency<IEventsSheetsService>()
             }
         }
 
+        public async Task ShowEvent(CommandContext context, InteractivityModule interactivity)
+        {
+            await context.RespondAsync($"{context.Member.Mention} - what is the event key?");
```

Comment:
Might be worth adding a little note in brackets here (and everywhere we ask for the event key), that the event key can be found in the event list? 🙂